### ISSUE TITLE
Sustainable funding for CC Admin team 

### ISSUE THREAD

**Issue by stephen-rowan** [Catalyst-Circle/Catalyst-Prioritized-Problems#67]
- [ ] Should there be a permanent allocation of treasury to support admin of Catalyst governance ?
See discussion here 
- https://github.com/Catalyst-Circle/Catalyst-Circle-Coordination/issues/38

**Comment by dimitrifernando** [comment]
Yes.  I have no idea who came up with the thought that 300 ADA per month was reasonable for a 40 hours a month job. This while every other proposal has allocations of 40-60 USD per hour, sometimes going up to 100 USD per hour for doing work which has far less permanent significance than Catalyst work. I propose that an allocation of funds from the Treasury be done to the tune of 4000 USD worth of ADA per month to be paid for each CCv3 member and  for the Admin team a similarly reasoable amount based on how much time they have to allocate during the month for their work Also a retroactive payment be made for CCv2 and CCv1 members for their hard work and time spent.

I am in agreement to Stephen Rowans suggestion and am bringing up the above as a new issue
